staff/views/index.py

```diff
@@ -0,0 +1,80 @@
+import functools
+import itertools
+import operator
+
+from django.db.models import Q
+from django.template.response import TemplateResponse
+from django.utils.decorators import method_decorator
+from django.views.generic import View
+
+from jobserver.authorization import CoreDeveloper
+from jobserver.authorization.decorators import require_role
+from jobserver.models import Backend, User
+
+
+# configure searchable models here, each must have get_staff_url defined
+configured_searches = [
+    {"model": Backend, "fields": ["name", "slug"], "order_by": "name"},
+    {
+        "model": User,
+        "fields": ["first_name", "last_name", "username"],
+        "order_by": "username",
+    },
```

Comment:
Personal preference: I'd find this slightly easier to read if each key was on its own line.  You can get Black to do this by adding a comma before the closing brace.

Thanks to Black, I've missed arguing about style in PRs!

Reply:
Fixed in https://github.com/opensafely-core/job-server/pull/941